libc/src/__support/OSUtil/linux/aarch64/vdso.h

```diff
@@ -0,0 +1,47 @@
+//===---------- aarch64 vdso configuration ------------------------* C++ *-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+#include "src/__support/CPP/string_view.h"
+namespace LIBC_NAMESPACE {
+namespace vdso {
+// macro definitions
+#define LIBC_VDSO_HAS_RT_SIGRETURN
```

Comment:
Will these macros be available in the public headers, or they are only used in the internal implementations?

Reply:
they should only be used privately

Reply:
Seems only gettimeofday is tested/used. Please add tests for all of them, or remove them.

Reply:
vDSO symbols are platform dependent. Are you suggesting that we should "add on need"?

Reply:
Right, please add unit tests that are surrounded by ifdefs for these newly added defines.